layouts/partials/search-input-custom.html

```diff
@@ -0,0 +1,20 @@
+{{/* Revisit this if / when either of https://github.com/google/docsy/issues/2194 and https://github.com/google/docsy/pull/1512 are closed */}}
+{{ $lang := .Site.Language.Lang }}
+{{ $searchFile := printf "content/%s/search.md" $lang }}
```

Comment:
You seem to be doing here manually here what GetPage can do automatically. For details, see https://gohugo.io/methods/site/getpage/#multilingual-projects

Reply:
I am hesitant to change the logic in this PR as this layout is a copy-paste of what was removed from `search-input.html` and I want the review to be simple. But I can definitely look into it for a fresh PR.

I am raising #50090 to track this.

Reply:
+1 for a cleanup in a follow-up PR